c2corg_ui/static/partials/contexthelp/associate.html

```diff
@@ -0,0 +1,13 @@
+<p translate>Associate documents (e.g. routes, waypoints, books...etc)</p>
+<ul>
+<li><span translate> Enter a discriminatory part of the document's name. It is possible to search on multiple keywords. For a route, this includes the name of the summit(s) associated to the route.</span></li>
```

Comment:
In addition you don't need the ``<span>`` element since there is no other content in the parent ``<li>`` element. Using
```
<li translate>Some text to translate</li>
```
has the same effect and is shorter, reducing the size of data users will download.

Same problem in the other partials.